examples/sample.py

```diff
@@ -0,0 +1,45 @@
+#!/usr/bin/python3
```

Comment:
I think we could simply put this example in the readme, it's the first file ppl look at so more convenient.

Reply:
ok